src/table/utils/__tests__/styling-utils.spec.js

```diff
@@ -173,40 +173,40 @@ describe('styling-utils', () => {
         fontSize: 22,
         color: resolvedColor,
         padding: '11px 22px',
-        hoverBackgroundColor: ['#f4f4f4', '!important'],
-        hoverFontColor: ['', '!important'],
-        selectedCellClass: '',
+        hoverBackgroundColor: '#f4f4f4',
+        hoverFontColor: '',
+        selectedCellClass: 'unselected',
```

Comment:
i think that 'unselected' is better than 'undefined'

Reply:
yeah it was unintentional that it would actually add ´undefined´ as a class, let's have unselected instead